### ISSUE TITLE
Vector mask is transformed when document size is not a square

### ISSUE THREAD

**Issue by sasensi** [Agamnentzar/ag-psd#44]
Hi, first I would like to thank you for your library which I just tried and which seems to work great !

# Explanation
While working with it, I discovered that vector masks are only working as expected if the document size is a square.
If you make it a rectangle, then the vector shape is scaled in the rectangle direction.
In order to prove the point, I used one of your test case: `/test/read/round/data.json`.
Here is the default result:
![Capture d’écran 2021-01-26 181557](https://user-images.githubusercontent.com/11247504/105879672-c0749900-6002-11eb-8201-b2cc27fefd37.png)
But if you change the document height from `200` to `400`, here is the result:
![Capture d’écran 2021-01-26 181617](https://user-images.githubusercontent.com/11247504/105879718-cf5b4b80-6002-11eb-9b1b-e5fd03244e9c.png)

# Reproduction
The bug can easily be reproduced by using the following code and switching the `SHOW_ISSUE` variable value to compare both situations.
```javascript
import * as fs from 'fs';
import { writePsdBuffer } from 'ag-psd';

const SHOW_ISSUE = true;

const ok = {
  'width': 200,
  'height': 200,
  'children': [
    {
      'top': 22,
      'left': 15,
      'bottom': 190,
      'right': 185,
      'name': 'shape layer',
      'vectorFill': {
        'type': 'color',
        'color': {
          'r': 72.00000330805779,
          'g': 72.00000330805779,
          'b': 72.00000330805779,
        },
      },
      'vectorMask': {
        'paths': [
          {
            'open': false,
            'knots': [
              {
                'linked': true,
                'points': [
                  54.160380363464355,
                  23.000001907348633,
                  100,
                  23.000001907348633,
                  145.83961963653564,
                  23.000001907348633,
                ],
              },
              {
                'linked': true,
                'points': [
                  183.00000429153442,
                  60.16037464141846,
                  183.00000429153442,
                  105.9999942779541,
                  183.00000429153442,
                  151.8396258354187,
                ],
              },
              {
                'linked': true,
                'points': [
                  145.83961963653564,
                  188.99999856948853,
                  100,
                  188.99999856948853,
                  54.160380363464355,
                  188.99999856948853,
                ],
              },
              {
                'linked': true,
                'points': [
                  16.999995708465576,
                  151.8396258354187,
                  16.999995708465576,
                  105.9999942779541,
                  16.999995708465576,
                  60.16037464141846,
                ],
              },
            ],
          },
        ],
      },
    },
  ],
};

const ko = {
  ...ok,
  height: 400,
}


const psd = SHOW_ISSUE ? ko : ok;

const buffer = writePsdBuffer(psd);
fs.writeFileSync('my-file.psd', buffer);
```
I reduced the original test case to keep only the relevant data but the bug also occurs if you keep the integral data or if you read/edit/write an original Photoshop file.

# Environment
I'm running Node v14.15.2 on Windows 10 and Photoshop 20.0.4

**Comment by Agamnentzar** [comment]
Yeah, PSD file saves vector data relative to document size for some reason, so I must have made mistake with scaling somewhere, I'll check it out and try to make a fix

**Comment by sasensi** [comment]
Ok, great, thank you !

**Comment by Agamnentzar** [comment]
Released a fix as version `11.4.1`

**Comment by sasensi** [comment]
Awesome, thanks for the quick fix, I just tested it and it seems to work like a charm !
